### PR TITLE
feat: add sonar cloud properties for better control

### PR DESCRIPTION
**Pull request by sawood14012**
Adding Sonar Cloud Properties File for better control

https://issues.redhat.com/browse/STONEBLD-1940

### CODE REVIEW

**Review by chmeliik — COMMENTED** [review]
We don't need the other properties like we have in build-service? https://github.com/redhat-appstudio/build-service/blob/main/.sonarcloud.properties

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml,
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Exclusions for copy-paste detection
+sonar.cpd.exclusions=**/*test*
```
Comment (chmeliik) [inline .sonarcloud.properties:9]: Could you add the missing newline?

**Comment by sawood14012** [comment]
> We don't need the other properties like we have in build-service? https://github.com/redhat-appstudio/build-service/blob/main/.sonarcloud.properties

I had added those just to identify those are not required i am planning to remove them from build service also

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Pull request merged**